src/structures/mode.jl

```diff
@@ -279,6 +280,36 @@ Returns the directions of transmission mode `tm`.
 """
 directions(tm::TransmissionMode) = tm.directions
 
+"""
+    has_emissions(tm::TransmissionMode)
+
+Returns whether there are emissions associated with transmisson mode `tm`.
+Default behaviour is no emissions.
+"""
+has_emissions(tm::TransmissionMode) = false
+
+"""
+    emit_resources(m::TransmissionMode)
+
+Returns the types of emisssions associated with transmission mode `tm`.
+"""
+emit_resources(tm::TransmissionMode) = EMB.ResourceEmit[]
+
+"""
+    emission(tm::TransmissionMode, p::EMB.ResourceEmit)
```

Comment:
Is this function necessary or do you foresee a future use?

Reply:
Not entirely, but good to have it included anyhow. There are situations where it is beneficial to pass the complete time profile, at least I could consider situations. We have methods like that for all other functions with time profiles included, so I would keep it here as well.